xiaogpt/bot/edgegpt_bot.py

```diff
@@ -0,0 +1,28 @@
+import asyncio
+
+from EdgeGPT import Chatbot, ConversationStyle
+from rich import print
+
+from xiaogpt.utils import split_sentences
+
+
+class EdgeGPTBot:
+    def __init__(self, proxy=None, cookiePath: str = ""):
+        self.history = []
+        self._bot = Chatbot(cookiePath=cookiePath, proxy=proxy)
+
+    async def ask(
+        self,
+        query,
+        wss_link="wss://sydney.bing.com/sydney/ChatHub",
+        conversation_style=ConversationStyle.creative,
+        **options,
+    ):
+        completion = await self._bot.ask(
+            prompt=query, wss_link=wss_link, conversation_style=conversation_style
```

Comment:
```suggestion
            prompt=query, wss_link=wss_link, conversation_style=conversation_style, **options
```

以便用户可以通过gpt_options覆盖默认值